opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -371,8 +375,12 @@ message DoubleDataPoint {
   // 1970.
   fixed64 time_unix_nano = 3;
 
-  // value itself.
-  double value = 4;
+  // The value itself.  A point is considered invalid when one of the recognized
+  // value fields is not present inside this oneof.
+  oneof value {
+    double as_double = 4;
+    double as_int = 6; 
```

Comment:
I like that you already transformed this in double :) Please fix, this should be fixed64 to ensure always 8 bytes

Reply:
heh, sorry. it's sfixed64, right?